### ISSUE TITLE
same alert SMS to multiple Numbers at the same time

### ISSUE THREAD

**Issue by ntassabhji** [ericallenpaul/Zabbix-Twilio.sh#10]
Hi, 

am not sure i unserstand the script funktionality pretty well, what am looking for is to have an SMS sent to multiple numbers at the same time whenever a trigger ist triggered.

I have everything configured, and sending the ALert SMS to a single number works fine, but i didnt figure out to send to multiple Numbers.

well appreciated

**Comment by ericallenpaul** [comment]
Yep this was not designed to send to multiple numbers at the same time. It just picks up the file and send to the number in that file. You would have to make some changes to the scripts for that to work.



Sent from my Galaxy



-------- Original message --------
From: ntassabhji ***@***.***>
Date: 10/15/24 6:52 AM (GMT-07:00)
To: "ericallenpaul/Zabbix-Twilio.sh" ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ericallenpaul/Zabbix-Twilio.sh] same alert SMS to multiple Numbers at the same time (Issue #10)


Hi,

am not sure i unserstand the script funktionality pretty well, what am looking for is to have an SMS sent to multiple numbers at the same time whenever a trigger ist triggered.

I have everything configured, and sending the ALert SMS to a single number works fine, but i didnt figure out to send to multiple Numbers.

well appreciated
